react-app/src/native/components/common/TeamPicker.js

```diff
@@ -48,9 +48,10 @@ class TeamPicker extends React.Component {
           >
             <Picker.Item key={0} label={label} value={''} />
             {
+              //console.log("curTeam = " + JSON.stringify(teams)),
               teams.filter((curTeam) => curTeam.id != excludeTeamId)
                 .map((curTeam) => {
-                  return <Picker.Item key={curTeam.id} label={curTeam.name} value={curTeam.id} />
+                  return <Picker.Item key={curTeam.id} label={curTeam.name + " - " + getDay(curTeam.scheduleLink)} value={curTeam.id} />
```

Comment:
I can't say if this would work for sure but the team weekday should be obtained through the league. If that's not coming back with the API i'll need to update it.